framework/src/main/java/org/checkerframework/framework/type/typeannotator/DefaultQualifierForUseTypeAnnotator.java

```diff
@@ -42,6 +45,8 @@ public DefaultQualifierForUseTypeAnnotator(AnnotatedTypeFactory typeFactory) {
         TreeUtils.getMethod(NoDefaultQualifierForUse.class, "value", 0, processingEnv);
   }
 
+  // TODO: Why isn't there a visitPrimitive()?  There can be default types for primitives.
```

Comment:
Beacause `@DefaultQualifierForUse` is an annotation the goes on a type declaration.  There's no way to add annotations to the declaration of primitives. (Defaults for primitives are add via the meta-annotation @DefaultFor. And those defaults are added elsewhere.)